chart/assets/operations/instance_groups/diego-cell.yaml

```diff
@@ -92,8 +92,25 @@
   path: /instance_groups/name=diego-cell/jobs/name=garden-cni/properties/cni_plugin_dir?
   value: /var/vcap/data/silk-cni/bin
 
-- type: remove
+- type: replace
   path: /instance_groups/name=diego-cell/jobs/name=silk-cni/properties/dns_servers?
+  value:
+  # The bosh-dns.<namespace>.svc gets resolved to the service ClusterIP it points to and then
+  # replaced with its value when rendering the silk-cni configuration file. This functionality is
+  # provided by a patch on a pre-render script.
+  - {{ printf "apps-dns.%s.svc" .Release.Namespace | quote }}
```

Comment:
Could we assume that the service IP address is in its environment variables?

Reply:
We could, but this is more explicit? I don't see any real advantage of using the environment variable. Both approaches would need a patch.